go.mod

```diff
@@ -4,13 +4,18 @@ go 1.13
 
 require (
 	github.com/coreos/prometheus-operator v0.41.1
-	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/logr v0.2.1
+	github.com/go-logr/zapr v0.2.0 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
+	github.com/openshift/api v0.0.0-20200917102736-0a191b5b9bb0 // release-4.5
 	github.com/operator-framework/operator-lib v0.1.0
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/client-go v0.18.6
+	golang.org/x/net v0.0.0-20201002202402-0a1ea396d57c // indirect
+	k8s.io/api v0.19.2
+	k8s.io/apimachinery v0.19.2
+	k8s.io/client-go v0.19.2
+	k8s.io/klog/v2 v2.3.0 // indirect
```

Comment:
These version updates are probably not needed. The `github.com/openshift/api v0.0.0-20200917102736-0a191b5b9bb0 // release-4.5` module should work with existing package versions.